wled00/wled.h

```diff
@@ -432,12 +433,13 @@ WLED_GLOBAL unsigned long lastMqttReconnectAttempt _INIT(0);  // used for other
     #define MQTT_MAX_TOPIC_LEN 32
   #endif
   #ifndef MQTT_MAX_SERVER_LEN
-    #define MQTT_MAX_SERVER_LEN 32
+    #define MQTT_MAX_SERVER_LEN 52
   #endif
 WLED_GLOBAL AsyncMqttClient *mqtt _INIT(NULL);
 WLED_GLOBAL bool mqttEnabled _INIT(false);
 WLED_GLOBAL char mqttStatusTopic[40] _INIT("");            // this must be global because of async handlers
-WLED_GLOBAL char mqttDeviceTopic[MQTT_MAX_TOPIC_LEN+1] _INIT("");         // main MQTT topic (individual per device, default is wled/mac)
+WLED_GLOBAL char mqttDeviceTopic[MQTT_MAX_TOPIC_LEN+1] _INIT("");         // main MQTT topic (individual per device, default is lights/mac)
+WLED_GLOBAL char mqttResponseTopic[MQTT_MAX_TOPIC_LEN + 1] _INIT("");      // MQTT response topic (individual per device, default is lights/mac/r)
```

Comment:
Best practice for service-like calls in a pub-sub framework is to accept the response topic as part of the request.  Apart from removing any possibility of a misconfigured server, it allows the client to specify a unique ephemeral topic for each transaction, ensuring that you don't ever accidentally receive a reply from some other client's service call.